openedx/core/djangoapps/credentials/utils.py

```diff
@@ -64,3 +64,34 @@ def get_user_program_credentials(user):
         programs_credentials_data = get_programs_for_credentials(user, programs_credentials)
 
     return programs_credentials_data
+
+
+def get_programs_credentials(user, category=None):
+    """Return program credentials data required for display.
+
+    Given a user, find all programs for which certificates have been earned
+    and return list of dictionaries of required program data.
+
+    Arguments:
+        user (User): user object for getting programs credentials.
+        category(str) : program category for getting credentials.
+
+    Returns:
+        list of dict, containing data corresponding to the programs for which
+        the user has been awarded a credential.
+    """
+    programs_credentials = get_user_program_credentials(user)
+    credentials_data = []
+    for program in programs_credentials:
+        if program.get('category') == category:
```

Comment:
This function should return all program credentials when called with the default category (`None`). As it stands, nothing will be returned. You could do:
```py
```